src/components/input-dropdown/input-dropdown.component.less

```diff
@@ -9,6 +9,12 @@ ux-input-dropdown {
         white-space: nowrap;
         overflow: hidden;
     }
+
+    &.ux-select-disabled .ux-select-container .ux-select-icons {
+        .ux-select-icon, .ux-select-icon {
+            color: #b3b3b3;
```

Comment:
The select component uses #d8d9db for the disabled icon, perhaps we can use the same here?